partials/java-versions.md

```diff
@@ -2,10 +2,9 @@
 
 Simply set the environment variable **CC_JAVA_VERSION** to the version you want
 
-Accepted values are `7` to `15` or `graalvm-ce` (for GraalVM 21.0.0.2, based on OpenJDK 11.0).
```

Comment:
We might want to state that we're following upstream by supporting LTS version + latest non-LTS